application/app/Rules/UserFullNameRule.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+
+class UserFullNameRule implements ValidationRule
+{
+    private const USER_NAME_REGEX = '/^[a-zõäöüšž\-]+(\s[a-zõäöüšž\-]+)$/iu';
```

Comment:
**[non-blocker]** Hmm, is it confirmed that we won’t to allow characters used in non-Estonian names? If this is confirmed with analysis, no problem, but I imagine there might probably be users with names such as "Radwan Dąbrowski-Żądło" or "Jānis Bērziņš".